.expeditor/verify.pipeline.yml

```diff
@@ -0,0 +1,44 @@
+steps:
+
+#########################################################################
+  # Lint Tests
+#########################################################################
+
+- label: lint-terraform
+  command:
+    - cd /workdir/terraform
+    - make lint
+  expeditor:
+    executor:
+      docker:
+        privileged: false
+
+- label: lint-chef-server-deploy-cookbook-cookstyle
+  command:
+    - cd /workdir/cookbooks/chef-server-deploy
+    - chef exec cookstyle
+  expeditor:
+    executor:
+      docker:
+        privileged: false
+
+- label: chef-server-deploy-cookbook-foodcritic
+  command:
+    - cd /workdir/cd cookbooks/chef-server-deploy
+    - chef exec foodcritic . -t ~supermarket
+  expeditor:
+    executor:
+      docker:
+        privileged: false
+
+- label: license-scout
+  command:
+    - cd /workdir
+    - .expeditor/license_scout.sh
+  expeditor:
+    executor:
+      docker:
+        privileged: false
+        environment:
+            - OCTOKIT_ACCESS_TOKEN 
```

Comment:
We won't be able to access this secret in the public infrastructure.

Reply:
ah.. good point